draft-pda-protocol.md

```diff
@@ -571,6 +580,12 @@ and an empty body. Malformed requests are handled as described in
 {{errors}}. Clients SHOULD NOT upload the same measurement value
 in more than one report if the leader responds with status 200 and an empty body.
 
+The leader responds to requests with out-of-date `HpkeConfig.id` values, indicated
+by `EncryptedInputShare.config_id`, with status 400 and an error of type
```

Comment:
`HpkeConfig.id` is a uint8, meaning that any server can have 256 different keys, ever. If the lifetime of any HPKE key is on the order of days (which is what I infer from the suggested `Cache-Control` value above), and we don't allow recycling of `HpkeConfig.id`, then won't servers run out of key configs in just a few years? Which is a roundabout way of asking: should `HpkeConfig.id` be bigger than 8 bits?

Reply:
That seems like probably the right approach, though I suppose we could recycle.

Reply:
Indeed -- the current design requires aggregators to recycle. Since this is orthogonal to caching the config, let's file an open issue to address separately?

Reply:
(We had an issue tracking HpkeConfig updates, so I just updated that issue accordingly.)